cmd/podman/libpodruntime/runtime.go

```diff
@@ -73,29 +83,31 @@ func GetRuntime(c *cli.Context) (*libpod.Runtime, error) {
 	// TODO flag to set libpod static dir?
 	// TODO flag to set libpod tmp dir?
 
-	if c.GlobalIsSet("cni-config-dir") {
-		options = append(options, libpod.WithCNIConfigDir(c.GlobalString("cni-config-dir")))
+	if c.Flags().Changed("cni-config-dir") {
+		options = append(options, libpod.WithCNIConfigDir(c.GlobalFlags.CniConfigDir))
 	}
-	if c.GlobalIsSet("default-mounts-file") {
-		options = append(options, libpod.WithDefaultMountsFile(c.GlobalString("default-mounts-file")))
+	if c.Flags().Changed("default-mounts-file") {
+		options = append(options, libpod.WithDefaultMountsFile(c.GlobalFlags.DefaultMountsFile))
 	}
-	if c.GlobalIsSet("hooks-dir") {
-		options = append(options, libpod.WithHooksDir(c.GlobalStringSlice("hooks-dir")...))
+	if c.Flags().Changed("hooks-dir") {
+		options = append(options, libpod.WithHooksDir(c.GlobalFlags.HooksDir...))
 	}
 
 	// TODO flag to set CNI plugins dir?
 
-	// Pod create options
-	if c.IsSet("infra-image") {
-		options = append(options, libpod.WithDefaultInfraImage(c.String("infra-image")))
-	}
-
-	if c.IsSet("infra-command") {
-		options = append(options, libpod.WithDefaultInfraCommand(c.String("infra-command")))
-	}
+	// TODO I dont think these belong here?
```

Comment:
Should these be moved to the pod modules?

Reply:
maybe? it doesnt seem like they are being used? leaving for now and will follow up it